hack/test-update-storage-objects.sh

```diff
@@ -159,7 +160,9 @@ killApiServer
 kube::etcd::stop
 TARGET_STORAGE="etcd3" \
   DATA_DIRECTORY="${ETCD_DIR}" \
+  ETCD=$(which etcd) \
   ETCDCTL=$(which etcdctl) \
+  ATTACHLEASE="${KUBE_OUTPUT_HOSTBIN}/etcd" \
```

Comment:
Actually no - see this comment:
https://github.com/kubernetes/kubernetes/pull/31242/files#r76370912